eks-distro-base/Dockerfile.minimal

```diff
@@ -25,43 +28,52 @@ RUN yum upgrade -y && \
     yum install -y yum-utils
 
 ENV DOWNLOAD_DIR /tmp/download
+ENV NEWROOT /newroot
+ENV FAKE_INSTALLS bash glibc p11-kit p11-kit-trust
 
-# there is a post install script for ca-certificates handled down below
-# there is a post install script for setup, it appears to just remove temp files
-RUN yumdownloader --destdir=$DOWNLOAD_DIR \    
-    ca-certificates \
-    setup \
-    system-release \
-    tzdata
-
-WORKDIR /newroot
+COPY scripts/ /usr/bin
 
-RUN for rpm in $DOWNLOAD_DIR/*.rpm; do rpm2cpio $rpm | cpio -idmv; done && \
-    rm -rf usr/share/man usr/share/doc && \
-    mkdir -p /newroot/{bin,boot,lib,root,run,sbin,var} && \
-    mkdir -p /newroot/var/{backups,cache,lib,local,lock,log,run,spool,tmp} && \
-    # cert files created by post install of ca-certs
-    cp -rf /etc/pki/ca-trust/extracted /newroot/etc/pki/ca-trust
+WORKDIR $NEWROOT
 
 # non root user
-RUN mkdir -p /newroot/home/nonroot && \
-    chown 65532:65532 /newroot/home/nonroot
-
-# tmp directory with correct permissions
-RUN mkdir -p -m 1777 /newroot/tmp
-
-COPY files/ /newroot
+RUN mkdir -p $NEWROOT/home/nonroot && \
+    chown 65532:65532 $NEWROOT/home/nonroot && \
+    # tmp directory with correct permissions
+    mkdir -p -m 1777 $NEWROOT/tmp
+
+RUN set -x && \
+    # setup rpm + yum
+    yumdownloader --destdir=$DOWNLOAD_DIR \
+        system-release && \
+    rpm -ivh --nodeps --root $NEWROOT $DOWNLOAD_DIR/system-release*.rpm && \
+    # "install" excluded deps we do not want in final image
+    yumdownloader --destdir=/tmp/exclude -x "*.i686"  \
+        $FAKE_INSTALLS && \        
+    rpm -ivh --justdb --nodeps --root $NEWROOT /tmp/exclude/*.rpm && \
+    yum --installroot $NEWROOT install -y \
+        basesystem \
+        filesystem \        
+        setup \        
+        tzdata && \    
+    # postinstall scriptlet for ca-certs will fail, manually extract and copy
+    # key files from host
+    yumdownloader --destdir=$DOWNLOAD_DIR \
+        ca-certificates && \
+    rpm -ivh --root $NEWROOT --justdb $DOWNLOAD_DIR/ca-certificates*.rpm && \
+    for rpm in $DOWNLOAD_DIR/ca-certificates*.rpm; do rpm2cpio $rpm | cpio -idmv; done && \
+    # cert files created by post install of ca-certs
+    update-ca-trust && \
+    cp -rf /etc/pki $NEWROOT/etc && \
+    rpm --root $NEWROOT --nodeps --justdb -e $FAKE_INSTALLS && \
+    cleanup "base"
 
-# Generate rpm db so image scanners can detect out of date packages
-RUN rpm --initdb --dbpath /newroot/var/lib/rpm && \
-    rpm -ivh --nodeps --justdb --dbpath /newroot/var/lib/rpm $DOWNLOAD_DIR/*.rpm && \
-    rpm --dbpath /newroot/var/lib/rpm -qa | sort
+COPY files/ $NEWROOT
 
 
 ################# BASE ########################
 FROM scratch as base
 ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
-ENV SSL_CERT_FILE=/etc/ssl/certs/ca-certificates.crt
```

Comment:
this was the wrong path and this var isnt needed anyway.  golang will automatically scan a list of known paths which include where our bundle exists.